Assets/HotmaxEditorVR/activeProp.cs

```diff
@@ -93,8 +93,17 @@ private void runPropStateMachine()
                     init.deletePanel.SetActive(true);
                 }
                 break;
-            case stateManager.editorModes.openMenuMode:
-                decorators["scaleControl"].enabled = true;
+            case stateManager.editorModes.spawnMenuMode:
```

Comment:
change spawnMenuMode back to OpenMenuMode - and remove case for propertiesMenuMode